codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -262,9 +262,6 @@ public void closeStream(final Http2Stream stream, ChannelFuture future) {
         future.addListener(new ChannelFutureListener() {
           @Override
           public void operationComplete(ChannelFuture future) throws Exception {
-            // Deactivate this stream.
-            connection().deactivate(stream);
```

Comment:
Was there a reason why the deactivation and state change was decoupled?  Was this so priority events would still be distributed to streams in this state?